Changes

```diff
@@ -1,5 +1,9 @@
 LIST OF CHANGES FOR NPG-QC PACKAGE
 
+ - SeqQC
+   - tag metrics as first check shown in summary table
+   - showing tag index detail for tag metrics check in summary table
```

Comment:
Please consider changing to a more precise description:
- tag metrics column moved to be next to the column with tag index and tag sequence
- show deplexig percent for individual tags in teh tag metrics column